.github/workflows/assign-issues.yaml

```diff
@@ -0,0 +1,17 @@
+name: Issue assignment
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: 'Auto-assign issue to Pulumiverse board'
+        uses: pozil/auto-assign-issue@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          assignees: simenandre,ringods,dirien,usrbinkat,rebelopsio
```

Comment:
Can you make `Environment variables` out of this maybe? Avoids to change it, or do you want it explicit in the code?

Reply:
That might make for an improvement by having this configured via the Pulumi based Github infra setup.